app/generic/connections.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+from fastapi import APIRouter, Depends
+from aries_cloudcontroller import AriesAgentControllerBase
+
+from dependencies import member_agent
+
+logger = logging.getLogger(__name__)
+
+
+router = APIRouter(prefix="/generic/connections", tags=["connections"])
+
+
```

Comment:
Should these calls be limited to only the member_agent?

Reply:
hmm I was wondering the same. Should cover most cases though imo. In the end everything you wanna connect to should be a sub wallet (except for YOMA wallet itself maybe?). My take on this is we can always create these endpoints for/extend them to whatever extra case we have. This should be the general use case though. That said if you think I'm wrong here or missing something please do say